test_runner/src/main/kotlin/ftl/client/google/AndroidCatalog.kt

```diff
@@ -26,19 +26,26 @@ object AndroidCatalog {
             .setProjectId(projectId)
             .executeWithRetry()
             .androidDeviceCatalog
+            .filterDevicesWithoutSupportedVersions()
     }
 
+    private fun AndroidDeviceCatalog.filterDevicesWithoutSupportedVersions() =
+        setModels(models.filterNotNull().filter { it.supportedVersionIds?.isNotEmpty() ?: false })
+
     fun getModels(projectId: String): List<AndroidModel> = deviceCatalog(projectId).models.orEmpty()
 
     fun supportedVersionsAsTable(projectId: String) = fetchAndroidOsVersion(projectId).toCliTable()
 
-    fun describeSoftwareVersion(projectId: String, versionId: String) = fetchAndroidOsVersion(projectId).getDescription(versionId)
+    fun describeSoftwareVersion(projectId: String, versionId: String) =
+        fetchAndroidOsVersion(projectId).getDescription(versionId)
```

Comment:
I would prefer:
```
fun describeSoftwareVersion(
    projectId: String, 
    versionId: String
) = fetchAndroidOsVersion(projectId).getDescription(versionId)
```
but this is just a suggestion, please check other places

Reply:
Ah sorry, it was merged before I implemented changes. Will do it within another PR